Tasks/AzureFunctionAppV1/operations/AzureAppServiceUtility.ts

```diff
@@ -122,9 +122,10 @@ export class AzureAppServiceUtility {
 
     public async getKuduService(): Promise<Kudu> {
         var publishingCredentials = await this._appService.getPublishingCredentials();
+        var accessToken = await this._appService._client.getCredentials().getToken();
         if(publishingCredentials.properties["scmUri"]) {
             tl.setVariable(`AZURE_APP_SERVICE_KUDU_${this._appService.getSlot()}_PASSWORD`, publishingCredentials.properties["publishingPassword"], true);
```

Comment:
>_PASSWORD [](http://example.com/codeflow?start=80&length=9)

how is this env variable consumed?  Does it and whatever consuming it need to be updated to take in the accessToken?

Reply:
I have removed this. This was making sure that the logs did not expose the password from the Publishing profile.

Reply:
@FinVamp1 We are not logging the token by any change, right?

Reply:
That's right. I am not logging the Token anywhere.